samples/meetings-context-app/csharp/README.md

```diff
@@ -0,0 +1,90 @@
+---
+page_type: sample
+description: This sample shows the contents of meeeting tab context object in a meeting tab and meeting context details where user can see the details of participants and meeting via bot.
```

Comment:
Description is kind of messy
This sample shows the contents of meeeting tab context object in a **meeting tab and meeting context details** where user can see the details of participants and meeting via bot.

Please refactor it, if we can.

Also, spelling mistake here "**meeeting** "